.github/workflows/dependent-tests.yml

```diff
@@ -2,9 +2,9 @@ name: Dependent tests
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, feature/** ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, feature/** ]
```

Comment:
This necessary?

Reply:
We want to run dependent tests on features branches, so this would be correct imo